### ISSUE TITLE
Empty #[typestate] module should not error

### ISSUE THREAD

**Issue by jquesada2016** [rustype/typestate-rs#24]
This issue is purely cosmetic and it's just a nice-to have.

I think this can be helpful in situations where one is quickly prototyping on an API design and want to state the intent that a module is meant to be a `typestate` implementation.

For example, the following currently errors, stating there's a missing automata.

```rust
#[typestate]
mod my_state {}
```

**Comment by jmg-duarte** [comment]
I don't feel very comfortable with your "solution", but the suggestion makes a lot of sense. 
What about reaching for middleground? Maybe something like:
```rust
#[typestate]
mod my_state {
  TODO!()
}
```
Or other similar syntax? Maybe even:
```rust
#[typestate(ignore)] // this should "deactivate" the macro
mod my_state {}
```

**Comment by jquesada2016** [comment]
I really like the first one:
```rust
#[typestate]
mod my_state {
  TODO!()
}
```

Just that I'd make it from `TODO!()` to `todo!()` to make it consistent with `std`.

**Comment by jmg-duarte** [comment]
> Just that I'd make it from `TODO!()` to `todo!()` to make it consistent with `std`.

Good point! I think that I'll declare a func-like proc macro for that, so that people can do:
```rust
typestate::todo!()
```
